src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -28,58 +30,54 @@ class JobRequest(object):
     """
     A small wrapper for job comm channel request data.
     This generally comes in the form of a packet from the kernel Comm object.
-    It's expected to be a dict of the format:
+    It is expected to be a dict of the format:
     {
         content: {
             'comm_id': <some string>,
             'data': {
-                'request_type': effectively, the function requested.
+                'request_type': job function requested
                 'job_id': optional
                 'job_id_list': optional
+                'batch_id': optional
             }
         }
     }
 
-    This little wrapper fills 2 roles:
+    This little wrapper fills some roles:
```

Comment:
"some" sounds strange -- can you change it to "two"?

Reply:
I thought "some" was more a scaffold type documentation, since I was just deleting obsolete documentation and smoothing over the difference